docs/docsite/rst/plugins/test.rst

```diff
@@ -23,7 +23,7 @@ You can add a custom test plugin by dropping it into a ``test_plugins`` director
 Using test plugins
 -------------------
 
-You can use tests anywhere you can use templating in Ansible: in a play, in variables file, or in a Jinja2 template for the :ref:`template <template_module>` module. For more information on using test plugins, see :ref:`playbooks_tests`.
+You can use tests anywhere you can use templating in Ansible: in a play, in a variables file, or a Jinja2 template for the :ref:`template <template_module>` module. For more information on using test plugins, see :ref:`playbooks_tests`.
```

Comment:
maybe it's just personal preference but I would use "in a" for all three things here. "in a play, in a file, in a template".